info/release-notes/20.after8.0-readme.txt

```diff
@@ -3,18 +3,24 @@
 
 TODO's
 ======  
-             
 
 
 New
 ===
 
-* 
+* Elapsed Time - Recorded Time - Paused Time (Frederic) https://github.com/wolfgang-ch/mytourbook/pull/222
+ - The recorded/paused time are imported from the following formats Garmin FIT, Suunto (Ambit 2 xml, Ambit 2 sml, Suunto Spartan/9 .json.gz), SportTracks Fitlog, Garmin TCX
+ - Added the ability to re-import, for existing tours, only the timer pauses (and hence also the recorded time).
+ - The new tour times (Recorded & Paused Times) are available for displau in the TourDataEditorView, TourBookView, Calendar view & Statistics View
+ - 2D Map: Added the ability to display/hide the location and duration of the tour pauses
```

Comment:
The readme should show the old and new time names, similar like in this PR comment

> New time data:
>
>    Elapsed time (formerly Recording Time)
>    Recorded Time
>    Paused Time (= Elapsed Time - Recorded Time)
>
>Existing time data:
>
>    Moving Time
>    Break Time ( = Elapsed Time - Moving Time)

Reply:
Fixed